src/main/java/bio/terra/service/snapshotbuilder/utils/constants/ConceptAncestorConstants.java

```diff
@@ -0,0 +1,9 @@
+package bio.terra.service.snapshotbuilder.utils.constants;
+
+public final class ConceptAncestorConstants {
```

Comment:
I suppose you don't need the `Constants` this is something that I thought about, I personally think it would read nicely to see the `Constants` when getting a term, to know that the variable is a constant. But since the value is in all capital letters it can already be assumed that the value is a constant.

Reply:
Yeah, I think the capital letters are enough there.